Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -360,3 +358,26 @@ extension PlaceCarouselViewController: LocationProvider {
     }
 }
 
+extension PlaceCarouselViewController: PlacesControllerDelegate {
+    func placesControllerNoPlacesFound(_ controller: PlacesController, willRetry retry: Bool) {
+        if retry {
+            // TODO placeholder for the waiting state.
+            headerView.numberOfPlacesLabel.text = "Waiting"
+        } else {
+            self.places = []
```

Comment:
`self.placeCarousel.refresh()` is called at https://github.com/mozilla-mobile/prox/pull/149/files#diff-28319a278e726e589796867f78f1bfc0R57